src/PDE/CompFlowProblem/NLEnergyGrowth.h

```diff
@@ -0,0 +1,462 @@
+// *****************************************************************************
+/*!
+  \file      src/PDE/CompFlowProblem/NLEnergyGrowth.h
+  \author    F. Gonzalez
```

Comment:
Aren't we removing these to support a more traditional open source development model?